src/assets/Group2/Clock.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="39" height="39" viewBox="0 0 39 39" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Good, try to use SVG over PNGs in web dev. Simple reason is 
1. smaller size (faster app)
2. Scalable (like a pdf)
3. less chance for corruption.
4. You may go in and look at an image like a code, and change certain things such as color, etc